gnucash/gtkbuilder/dialog-customer.glade

```diff
@@ -210,7 +210,7 @@
                       <object class="GtkLabel" id="label2">
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
-                        <property name="label" translatable="yes">Identification</property>
+                        <property name="label" translatable="yes">Information</property>
```

Comment:
No, Name and Number are the Item which idenify the company. Information is too general, so it becomes meaningless